Makefile.inc

```diff
@@ -586,7 +586,7 @@ $(ECCTOOL):
 	cd $(objutil)/ffs && autoreconf -i && ./configure
 	+$(MAKE) -C $(objutil)/ffs
 
-$(SBSIGNTOOLS):
+$(SBSIGNTOOLS)/create-container:
```

Comment:
Maybe we should rather make SBSIGNTOOLS point to `create-container` and then call it directly instead of using script. We don't have support for Secure Boot yet anyway, so probably first call would be enough.

GitHub won't let me add a comment to line 1188, but I would like you to check if image built with just (instead of mentioned line):
```
./create-container -a $KEYLOC/hw_key_a.key -b $KEYLOC/hw_key_b.key -c $KEYLOC/hw_key_c.key \
                   -p $KEYLOC/hw_key_a.key \
                    --payload $PAYLOAD --imagefile $OUTPUT
```
boots on hardware. Of course, you would have to change the variables.

Reply:
This variant seems to work. In both cases output ends like this:
```
BS: BS_WRITE_TABLES run times (exec / console): 1 / 4 ms
CBFS: 'fallback/payload' not found.
Payload not l
```